ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -280,7 +280,7 @@ class WalletHomeScreen extends React.Component<Props, never> {
         ) : (
           <TransactionsList
             title={I18n.t("wallet.latestTransactions")}
-            totalAmount={I18n.t("wallet.total")}
+            totalAmount={I18n.t("wallet.amount")}
```

Comment:
Please also change the name of the prop.

Reply:
Done!